.gitignore

```diff
@@ -4,7 +4,6 @@
 .packages
 .dart_tool/
 build/
-packages
```

Comment:
Do you knwo why `packages` was previously in gitignore?

Reply:
this was added 8 years ago, so it might have been some Dart/Flutter specific thing that was still an issue back then. 

today `packages` is used a lot, e.g https://github.com/flame-engine/flame/